packages/neutrino-preset-web/index.js

```diff
@@ -67,14 +67,13 @@ module.exports = (neutrino) => {
   neutrino.use(htmlTemplate, neutrino.options.html);
   neutrino.use(namedModules);
   neutrino.use(compileLoader, {
-    include: [neutrino.options.source, neutrino.options.tests],
+    include: [neutrino.options.source, neutrino.options.tests, require.resolve('./polyfills.js')],
```

Comment:
To be honest didn't think about this "include". I see that it will not affect the build until somebody will not import exactly this file `./polyfills.js`.